app/src/main/java/com/infomaniak/mail/data/models/message/Body.kt

```diff
@@ -24,5 +24,4 @@ import kotlinx.serialization.Serializable
 class Body : EmbeddedRealmObject {
     var value: String = ""
     var type: String = ""
-    var subBody: String? = null
```

Comment:
Do we want to remove subBody ?

Reply:
It's unused.

Reply:
Can't be deserialized for now, will be handled when we want to support this feature.